src/test/java/io/confluent/connect/elasticsearch/integration/ElasticsearchIntegrationTestBase.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.connect.elasticsearch.integration;
+
+import static io.confluent.connect.elasticsearch.DataConverter.BehaviorOnNullValues;
+import static org.junit.Assert.assertEquals;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import io.confluent.connect.elasticsearch.DataConverter;
+import io.confluent.connect.elasticsearch.ElasticsearchClient;
+import io.confluent.connect.elasticsearch.IndexableRecord;
+import io.confluent.connect.elasticsearch.jest.JestElasticsearchClient;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+
+public class ElasticsearchIntegrationTestBase {
+
+  protected static final String TYPE = "kafka-connect";
+
+  protected static final String TOPIC = "topic";
+  protected static final int PARTITION = 12;
+  protected static final int PARTITION2 = 13;
+  protected static final int PARTITION3 = 14;
+  protected static final TopicPartition TOPIC_PARTITION = new TopicPartition(TOPIC, PARTITION);
+  protected static final TopicPartition TOPIC_PARTITION2 = new TopicPartition(TOPIC, PARTITION2);
+  protected static final TopicPartition TOPIC_PARTITION3 = new TopicPartition(TOPIC, PARTITION3);
+
+  private static final String DEFAULT_ES_VERSION = "7.0.0";
+  private static final String DEFAULT_DOCKER_IMAGE_NAME = "docker.elastic.co/elasticsearch/elasticsearch";
+  private static final String ELASTICSEARCH_VERSION_SYS_PROP = "elasticsearch.version";
+  private static final String ELASTICSEARCH_VERSION_ENV_PROP = "ELASTICSEARCH_VERSION";
+  private static final String ELASTICSEARCH_DOCKER_IMAGE_SYS_PROP = "elasticsearch.docker.image";
+  private static final String ELASTICSEARCH_DOCKER_IMAGE_ENV_PROP = "ELASTICSEARCH_DOCKER_IMAGE";
```

Comment:
Any chance we can be consistent with the other already released library and use `elasticsearch.image` and `ELASTICESEARCH_IMAGE` instead? I think we don’t need to include “Docker” in the name since that is pretty well assumed.